src/modules/interactive.js

```diff
@@ -0,0 +1,19 @@
+import { removve } from './functioning.js';
```

Comment:
How about declaring the variable as remove instead of remove for easier readability